src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+    private static final String DEFAULT_LOGIN = "bob";
+    private static final String DEFAULT_PASSWORD = "bob";
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login(DEFAULT_LOGIN, DEFAULT_PASSWORD);
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Login error. Params: login = {}",DEFAULT_LOGIN);
```

Comment:
looks good but we already have in logs our parameters for this method, but i think you can leave it like this